src/cbc_sdk/platform/vulnerability_assessment.py

```diff
@@ -815,6 +938,28 @@ def _run_async_query(self, context):
             if current >= self._total_results:
                 return self._doc_class(self._cb, initial_data=results)
 
+    def export(self):
```

Comment:
Would be good to have an example snippet of what to do with the returned object.  Either in line here, or added to the vuln guide.